packages/build/src/utils/json.ts

```diff
@@ -1,21 +1,18 @@
-import { promises as fs, readFileSync } from 'fs'
+import { readFile } from 'fs/promises'
 import { fileURLToPath } from 'url'
 
-import { PackageJson } from 'read-pkg-up'
+import type { PackageJson } from 'read-pkg-up'
+
+// We know how our package.json looks like, so we can be very specific with the type
+// and only add the properties we want to use
+export type RootPackageJson = { name: string; version: string }
 
 const ROOT_PACKAGE_JSON_PATH = fileURLToPath(new URL('../../package.json', import.meta.url))
 
-// TODO: Replace with dynamic `import()` once it is supported without
```

Comment:
I think you can still leave the comment. It's for: https://nodejs.org/dist/latest-v18.x/docs/api/esm.html#import-assertions